src/styles/index.scss

```diff
@@ -1,3 +1,120 @@
-body {
-  margin: 0;
+@import "./variables.scss";
+
+.card {
```

Comment:
according to https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md you need to create a separate file for each BEM block